airbyte-integrations/connectors/source-bigquery/metadata.yaml

```diff
@@ -22,13 +23,14 @@ data:
   dockerImageTag: 0.1.0
   dockerRepository: airbyte/source-bigquery
   githubIssueLabel: source-bigquery
-  icon: bigquery.svg
+  icon: icon.svg
   license: MIT
   name: Bigquery
-  releaseDate: TODO
+  releaseDate: 2024-06-17
```

Comment:
Why do we need this `releaseDate`?

Reply:
@bazarnov 
If I do not fill it the spec acceptance test fails